### ISSUE TITLE
package request: greetd

### ISSUE THREAD

**Issue by blaggacao** [colemickens/nixpkgs-wayland#254]
https://sr.ht/~kennylevinsen/greetd/
complement: https://github.com/apognu/tuigreet

**Comment by duairc** [comment]
Note that @luc65r has already done [most of the work to package this for NixOS](https://github.com/NixOS/nixpkgs/pull/102242), it just seems to blocked on [this PR](https://github.com/NixOS/nixpkgs/pull/102225) for now. There is also some discussion [here](https://github.com/NixOS/nixpkgs/issues/87603) and a preliminary [NixOS module](https://github.com/MetaDark/nur-packages/blob/greetd/modules/services/x11/display-managers/greetd/greetd.nix) in the NUR repository of @MetaDark.

(I've not tested any of the above.)

**Comment by blaggacao** [comment]
I think this is closing-worthy, then. Thanks for the pointers!
